src/era_5g_relay_network_application/era_5g_relay_network_application/client.py

```diff
@@ -282,65 +220,66 @@ def signal_handler(sig: int, frame: Optional[FrameType]) -> None:
-                callback = partial(
-                    callback_image,
-                    topic_type=topic_type,
-                    topic_name=topic_name_remapped,
-                )
+            
+            q = Queue(1)
+            channel_type = get_channel_type(compression, topic_type)
+            if channel_type in IMAGE_CHANNEL_TYPES:
+                worker = WorkerImageSubscriber(topic_name, topic_type, compression, node, q)
+                send_function = partial(send_image, event=f"topic/{topic_name}", client=client, channel_type=channel_type, can_be_dropped=True)
             else:
-                callback = partial(
-                    callback_others,
-                    topic_type=topic_type,
-                    topic_name=topic_name_remapped,
-                    compression=compression,
-                )
-
-            subs.append(node.create_subscription(topic_type_class, topic_name, callback, BEST_EFFORT))
-        for service_name, service_type in services:
-            service_type_class = ros_loader.get_service_class(service_type)
-            node.create_service(
-                service_type_class,
-                service_name,
-                partial(
-                    callback_service,
-                    service_name=service_name,
-                    service_type=service_type,
-                ),
-            )
-
-        tf_republisher: Optional[TFRepublisher] = None
-
-        if transforms:
-            tf_republisher = TFRepublisher(node, tf_callback)
-            for tr in transforms:
-                logger.info(f"Subscribing for: {tr}")
-                tf_republisher.subscribe_transform(*tr)
-
-        while rclpy.ok():
-            rclpy.spin_once(node, timeout_sec=1.0)
-
+                worker = WorkerSubscriber(topic_name, topic_type, compression, node, q)
+                send_function = partial(client.send_data, event=f"topic/{topic_name}", channel_type=channel_type, can_be_dropped=True)
+                
+            worker_socketio = WorkerSocketIO(q, send_function)
+            worker_socketio.daemon = True
+            worker_socketio.start()
+            
+            socketio_workers.append(worker_socketio)
+        
+        for service_name, service_worker in services_workers.items():
+            channel_name = f"service_request/{service_name}"
+            send_function = partial(client.send_data, event=channel_name, can_be_dropped=False)
+            worker_socketio = WorkerSocketIO(service_worker.request_queue, send_function)
+            worker_socketio.daemon = True
+            worker_socketio.start()
+            socketio_workers.append(worker_socketio)
+            
+
+        # create socketio worker for all transforms to be sent to the relay server
+        if transforms_to_listen:
+            tf_queue = Queue(1)
+            worker_tf = WorkerTF(transforms_to_listen, tf_queue, node)
+            send_function = partial(client.send_data, event="topic//tf")
```

Comment:
Is `//` intentional?

Reply:
Yes, the "topic/" is the prefix for all topic channels (most of the topics start with /, therefore, the double slash).